wasm-build/wordpress-data/web-publish.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+root_dir=../..
+dest_dir=$root_dir/dist-web
+
+cp ./docker-output/* $dest_dir/
+cp ./preload/wp-lazy-files.js $dest_dir/
```

Comment:
I think the paths are relative to the current working directory, which means the `cp` here wouldn't find `./preload/wp-lazy-files.js` since it's executed from the root repo while `./prelload` is a subdirectory of `wasm-build/php`.

Reply:
Wait, I'm wrong. This script is executed by npm with the correct cwd.